projects/fmcomms5/zcu102/system_bd.tcl

```diff
@@ -0,0 +1,13 @@
+
+source $ad_hdl_dir/projects/common/zcu102/zcu102_system_bd.tcl
+
+#set_property -dict [list CONFIG.PCW_EN_CLK2_PORT {1}] $sys_ps7
```

Comment:
After a second look, I think I know why you did that. But this raise up other questions, how your /common/fmcomms5_bd.tcl file looks like?